### PR TITLE
fix task list filter

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4791
https://tools.hmcts.net/jira/browse/EUI-4557

### Change description ###
Delay the execution on load tasks until filters have been persisted
Display the `Task` `created_date` on the Task Tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts [hunk 1]
```diff
@@ -86,6 +85,7 @@ export class TaskListFilterComponent implements OnInit, OnDestroy {
   public subscribeToSelectedLocations(): void {
     this.selectedLocationsSubscription = this.filterService.getStream(TaskListFilterComponent.FILTER_NAME)
       .pipe(
```
Comment (udaydenduluri33) [inline src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts:87]: please remove tap and console

Comment (kuda1992) [reply src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts:87]: Thanks. Forgot to clean that up.

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

**Review by udaydenduluri33 — APPROVED** [review]
@kuda1992 thanks for making the changes

**Pull request merged**